server/services/prisonerProfileService.ts

```diff
@@ -27,13 +28,21 @@ export default class PrisonerProfileService {
     let visitBalances = null
 
     if (convictedStatus !== 'Remand') {
-      visitBalances = await prisonApiClient.getVisitBalances(offenderNo)
+      visitBalances = (await prisonApiClient.getVisitBalances(offenderNo)) as BAPVVisitBalances
 
       if (visitBalances.latestIepAdjustDate) {
+        visitBalances.nextIepAdjustDate = format(
```

Comment:
I want to test this but at the same time, it's just testing a library which is already tested 🤷‍♂️